felix/wireguard/wireguard.go

```diff
@@ -282,6 +284,7 @@ func NewWithShims(
 		writeProcSys:         writeProcSys,
 		opRecorder:           opRecorder,
 		logCtx:               logCtx,
+		rateLimitedLogger:    lclogutils.NewRateLimitedLogger(lclogutils.OptInterval(4 * time.Hour)),
```

Comment:
Can we transfer the context from `logCtx` to this ratelimited logger some how?

Reply:
You might be able to do `NewRateLimitedLogger().WithFields(logCtx.Data)`

Reply:
Good point, will do that! Thanks for bringing this one up!